lib/rules/selector-pseudo-class-allowed-list/__tests__/index.mjs

```diff
@@ -233,6 +233,33 @@ testRule({
 			endLine: 1,
 			endColumn: 8,
 		},
+		{
+			code: '@page :first {}',
+			message: messages.rejected(':first'),
+			line: 1,
+			column: 7,
+			endLine: 1,
+			endColumn: 13,
+		},
+		{
+			code: '@page:blank:first {}',
+			warnings: [
+				{ message: messages.rejected(':blank'), line: 1, column: 6, endLine: 1, endColumn: 12 },
+				{ message: messages.rejected(':first'), line: 1, column: 12, endLine: 1, endColumn: 18 },
+			],
+			skip: true,
+		},
```

Comment:
see https://drafts.csswg.org/css-page/#example-691ff5b9
i.e. it seems like `walkAtRules` is buggy

Reply:
How is it buggy?
What happens, what do you expect instead?

Reply:
I am not really sure, I didn't investigate much.
I would expect `@page:blank:first {}` to be valid based on that link and hence that `walkAtRules` to recognizes it as an `atrule` node and not have to skip this test.

Reply:
Seems like a bug in PostCSS.

For: `@page:blank:first {}`

https://astexplorer.net/

<img width="408" alt="Screenshot 2024-12-07 at 09 34 36" src="https://github.com/user-attachments/assets/3592f0b6-1da4-4460-ae74-8db8d58e8ed5">


vs.

https://tabatkins.github.io/parse-css/example.html

<img width="381" alt="Screenshot 2024-12-07 at 09 34 53" src="https://github.com/user-attachments/assets/450899cc-e149-4de8-846d-af078e25c04e">

------

While `@media(color) {}` does produce the expected outcome.
Same for `@page#foo {}`.

But `@page.foo {}` and `@page&.foo` also seem wrong.

So I guess that the character set used by PostCSS to detect the end of the `name` is incorrect.

Reply:
I see.

We can move on with this PR and we will come back to unskip the test in the eventual dependabot PR that will update `postcss`.

Reply:
Just a last question, what about `walkAtRules(/^page/i, () => …` instead?

Reply:
I'd rather not :) Upstream bugs should be fixed upstream. And no one is writing `@page:first` without a separating space so there isn't much reason to add workarounds. Better to immediately fix it upstream.

Reply:
I am scared that minifier do write it like that though.

Reply:
No one should be linting the output of minifiers 🤔 
If anyone is affected by this (no one will be) then they can file an issue.
But as I said before, there is a clear path to fixing this, upstream, in PostCSS.